### PR TITLE
change settings page

### PR DESCRIPTION
**Pull request by devfake**

### CODE REVIEW

File: backend/app/Http/Controllers/SettingController.php [hunk 1]
```diff
@@ -149,9 +149,33 @@ public function settings()
...
+        'genre' => $settings->show_genre,
+        'date' => $settings->show_date,
+        'spoiler' => $settings->episode_spoiler_protection,
+      ];
+    }
+
+    /**
+     * Save new user options.
      */
-    public function changeSettings()
```
Comment (exane) [inline backend/app/Http/Controllers/SettingController.php:154]: Why "options"? Your code still uses settings und files are called "SettingsController" or "SettingsTest". Either everthing settings or options. IMO Settings sounds better.

File: client/app/components/Content/Item.vue [hunk 2]
```diff
@@ -42,11 +43,14 @@
 
     props: ['item', 'genre', 'date'],
 
+    created() {
+      this.saveNewRating = debounce(this.saveNewRating, 700);
```
Comment (exane) [inline client/app/components/Content/Item.vue]: Please no magic numbers. Use a variable that describes what it does.

File: client/app/components/Content/Settings/Misc.vue [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      checkUpdate() {
+        http(`${config.api}/check-update`).then(response => {
+          this.isUpdate = response.data;
+        });
+      },
+
+      fetchVersion() {
+        this.SET_LOADING(true);
+        http(`${config.api}/version`).then(response => {
+
```
Comment (exane) [inline client/app/components/Content/Settings/Misc.vue:67]: why line break here? you propably meant to use it between SET_LOADING and http.

File: client/app/components/Content/Settings/Options.vue [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+  import http from 'axios';
+  import debounce from 'debounce';
+
+  export default {
+    mixins: [Helper],
+
+    created() {
+      this.fetchOptions();
+      this.updateOptions = debounce(this.updateOptions, 700);
```
Comment (exane) [inline client/app/components/Content/Settings/Options.vue]: magic number again.

File: client/app/components/Content/Settings/User.vue [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+      })
+    },
+
+    methods: {
+      ...mapMutations([ 'SET_LOADING' ]),
+
+      fetchUserData() {
+        this.SET_LOADING(true);
+        http(`${config.api}/settings`).then(response => {
+
```
Comment (exane) [inline client/app/components/Content/Settings/User.vue:51]: line break?

File: client/resources/sass/components/_content.scss [hunk 6]
```diff
@@ -381,10 +381,48 @@ main {
...
+
+    &:active {
+      opacity: .6;
+    }
+
+    &.active {
+      color: $main2;
+      border-bottom: 2px solid $main2;
+
+        @include media(6) {
```
Comment (exane) [inline client/resources/sass/components/_content.scss]: whats up with the indention here?

File: backend/app/Http/Controllers/SettingController.php (hunk 1 above)
Comment (devfake) [reply backend/app/Http/Controllers/SettingController.php:154]: Yeah, you are right

**Pull request merged**